package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "dynamo-hosted-contents",
+  "name": "@dynamods/librarieJS",
```

Comment:
@Enzo707 Also it looks like there is no version info here so publish would not work correctly. You can use version 1.0.0